### ISSUE TITLE
Verify v0.1.0 — CI green on a4df504d + first real-model curve

### ISSUE THREAD

**Issue by stephenlutar2-hash** [szl-holdings/szl-quant-bench#1]
Verification gate for the v0.1.0 payload (commit `a4df504d2806349fe13470816afb0a7564883a2d`).
The suite was executed 8/8 green pre-push; the README curve was produced by
`run_curve` on a labeled synthetic fixture. This issue tracks independent
confirmation and the first measurement on real model logits. Close only with
evidence attached.

## Checklist

- [ ] CI run on `a4df504d` is green on Python 3.11 and 3.12 (Actions tab)
- [ ] Fresh clone: `pip install -e . pytest && python -m pytest tests/ -q` → 8 passed
- [ ] Demo: `python -m szl_quant_bench.harness` prints the fixture curve
      (16-bit cosine 1.0000 → 2-bit 0.5715), `chain_valid: true`
- [ ] First real-model curve: capture a logit batch from an estate model
      (e.g. `SZLHOLDINGS/SZL-Khipu-1.5B`) on a fixed prompt set, run
      `run_curve(logits, chain=chain)`, attach the receipt JSON here
- [ ] GGUF decision record: any 4-bit vs 8-bit claim references the receipted
      curve for the exact model revision, prompt set, and hardware used

## Non-negotiables

- The synthetic fixture curve in the README is a pipeline check, not evidence
  about any real model. Do not cite it as one.
- No cross-hardware or cross-revision comparisons in one table.
- No `MEASURED` claim without the emitted receipt JSON attached.

**Comment by stephenlutar2-hash** [comment]
Offline suite verified on this pass: `8 passed` under `tests/`.

Remaining: first real-model curve on owner logits. No fabricated bit-width numbers from this connector.

**Comment by stephenlutar2-hash** [comment]
### Operator-machine receipt — synthetic pipeline verified, real-model curve unavailable

Witnessed at exact `main` `25daa8751abb4c1ed0733ffa0f841fd5393ae118`.

- Exact-main CI [run 33891159203](https://github.com/szl-holdings/szl-quant-bench/actions/runs/33891159203): PASS on 3.11/3.12.
- Fresh checkout: 8 tests passed.
- Labeled synthetic fixture reproduced and chain verified; receipt `b477e8fe676b458e27a7c4327766ad3b6d4e33614a7090ae6e18840f83405604`. This is MODELED pipeline evidence only.
- OPEN/UNAVAILABLE: `SZLHOLDINGS/SZL-Khipu-1.5B` is a 3,246,659,806-byte provider artifact, but weights are not cached and neither torch nor an inference backend is installed. No real logits, hardware curve, or GGUF decision is claimed.

```json
{"chain_hash":"947f07a85f6bf9864110a6819e5066bb4c5204d6d1a548b673a5eba1dded1ca8","payload":{"checks":{"exact_main_ci":{"jobs":["test (3.11)","test (3.12)"],"run":33891159203,"state":"PASS"},"fresh_checkout_tests":{"count":8,"state":"PASS"},"real_model_logits":{"model":"SZLHOLDINGS/SZL-Khipu-1.5B","provider_bytes":3246659806,"state":"UNAVAILABLE","torch_available":false,"weights_cached":false},"synthetic_curve":{"chain_valid":true,"run_receipt":"b477e8fe676b458e27a7c4327766ad3b6d4e33614a7090ae6e18840f83405604","state":"MODELED"}},"honesty":"UNSIGNED_HONEST","issue":1,"machine":{"arch":"AMD64","cpu":"Intel Core Ultra 9 285H","gpu":["NVIDIA GeForce RTX 5050 Laptop GPU","Intel Arc 140T GPU"],"os":"Microsoft Windows 11 Home 10.0.26200","python":"3.12.14","ram_gb":31.59},"observed_at":"2026-09-04T16:47:00Z","repo":"szl-holdings/szl-quant-bench","schema":"szl.operator-gate-receipt.v1","source":{"branch":"main","commit":"25daa8751abb4c1ed0733ffa0f841fd5393ae118"}},"prev_hash":"b7e236d86f567040e79275a64912c6fcb171209448f56141c618467952d1a0d0"}
```

**Comment by stephenlutar2-hash** [comment]
Operational successor: PR #3 is merged at 675c03c882e5399e3c78861d71e109f14ba1ee5c, exact main CI33977908370 succeeds on Python3.11/3.12. Signed feature head19b59e70e89cdc0dc41f91bc0f0b370dec0b33e3 passed38tests on both versions and clean re-review. Version0.2 adds finite rectangular input/bit validation, input+provenance binding, immutable receipt snapshots, log-space KL, and a no-overwrite JSON CLI. Missing/wholly empty matrices remain BLOCKED.

This gate is reopened because the first estate-model logits and GGUF evaluation criteria remain unmet. Fresh2026-09-05 host observation: torch and transformers ARE now installed (supersedes yesterday's missing-library note), but SZL-Khipu cached weight files=0 and captured real logits remain UNAVAILABLE. Free physical memory is constrained by other work. No model load, weight-quantization comparison, GGUF decision, or real-model measurement is claimed. Source labels are explicit CALLER_DECLARED; a logit curve alone is not weight quantization evidence.

Local runtime observation receipt SHA256 ff5d371063068ec2b0f2e97979eba86fb0680000ad0fc673d4e756aa4cdebc4b. Original version-specific CI/test counts are historical; current successor source/CI evidence is above. Functional model-evidence criteria remain OPEN.

**Comment by stephenlutar2-hash** [comment]
Bounded convergence update: source repair is now open at PR #5, exact base `45b4a4ee53f77540cf712f966ffc75c4741f5bbe`, exact head `cb48ab6f5ab5bb4e14726b3178835c9b24ed3320`.

This closes one concrete evidence-contract gap in the issue checklist: the Python API could previously return `MEASURED` without emitting any receipt when the caller omitted `chain=`. PR #5 makes every `MEASURED` result carry a hash-chained receipt, preserves caller-owned chain behavior, adds regressions, and documents that the receipt does not upgrade `CALLER_DECLARED` provenance into authentication.

Native PR-event CI run `35809602614` is successful on that exact head for Python 3.11 and 3.12, including the full test suite and harness execution. The PR remains unmerged because current `main` is still unprotected and no effective repository ruleset is present; no independent review or DCO qualification is claimed.

Issue #1 remains open: this repair does not supply the requested first real-model curve, independent model execution evidence, or any GGUF/runtime qualification.
